models/e2node-1.x/api/gnmi_client.go

```diff
@@ -245,7 +246,7 @@ func (c *GnmiClient) Get_E2NodeIntervalsPdcpMeasReportPerUe(ctx context.Context,
 					Name: "intervals",
 				},
 				{
-					Name: "PdcpMeasReportPerUe",
+					Name: "SchedMeasReportPerUe",
```

Comment:
I'm not sure where the ordering comes from, I assume it's in the `ygot` generated structs as I'm using a list to store the generated methods but haven't investigated yet. I have https://jira.opennetworking.org/browse/AETHER-3550 to address that in this sprint. Will probably take that over before tackling the nested lists as it will make reviews much simpler.